pages/instructorQuestionManualGrading/instructorQuestionManualGrading.ejs

```diff
@@ -22,6 +22,32 @@
     </div>
     <div id="content" class="container">
       <div class="row">
+
+        <% if (typeof submission != null) { %>
+          <% if (submission.score == null && submission.manual_grading_user != null) { %>
+          <div class="col-lg-12 col-sm-12">
+            <div class="card mb-4">
+              <div class="card-header bg-warning text-blue">
+                <% if (authn_user.uid === user.uid) { %>
+                  You are currently grading this question
```

Comment:
I think we should skip this statement and only display something if someone else is currently grading the question.

Reply:
It might seem redundant, but the view is similar to the question page. This message helps a new user differentiate between a grading page. It can also help a new user understand what is happening behind the scenes while they are grading. Perhaps "You are currently grading this submission. Nobody else will be able to access this submission while you are grading."

It's a nice-to-have. If you want to minimize lines of code and logic, this can go.

Reply:
The right panel and perhaps heading info can be used to characterize the page as a grading page. We should save text for critical warning info indicating that the grading state of the question_instance has changed.

Reply:
or yes, as matt says, to indicate that someone else is grading the question.

Reply:
S

> The right panel and perhaps heading info can be used to characterize the page as a grading page. We should save text for critical warning info indicating that the grading state of the question_instance has changed.

Sounds like a good plan. That will happen then.

Reply:
Only left in to indicate when someone else is grading the question.